### PR TITLE
Enable continuous integration and inhibit pod warnings

### PR DESCRIPTION
**Pull request by dantheli**
This adds yaml files for [Travis-CI](http://travis-ci.org) and SwiftLint but does not actually run the SwiftLint script for build just yet (too many errors)

Also prevents CocoaPods from generating Xcode warnings in any of the pods during build.

**Please don't review yet!**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+osx_image: xcode9
+language: swift
+xcode_workspace: Clicker.xcworkspace
```
Comment (natashaarmbrust) [inline .travis.yml]: ^ W aren't clicker .. change to `Podcast`

File: PodcastTests/PodcastTests.swift [hunk 2]
```diff
@@ -20,17 +20,12 @@ class PodcastTests: XCTestCase {
...
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
-    }
-    
-    func testPerformanceExample() {
-        // This is an example of a performance test case.
-        self.measure {
-            // Put the code you want to measure the time of here.
-        }
+
+    func testKeys() {
```
Comment (natashaarmbrust) [inline PodcastTests/PodcastTests.swift:24]: when will this be run?

**Review by natashaarmbrust — APPROVED** [review]
LGTM .. when you merge in this PR please also include a comment describing changes developers should be aware of now that this PR is merged in

Whoops saw your comment not to review.

File: PodcastTests/PodcastTests.swift (hunk 2 above)
Comment (dantheli) [reply PodcastTests/PodcastTests.swift:24]: It is run automatically when the test suite is run

**Pull request merged**